src/components/SignUp.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import styled from "styled-components";
+import { Button } from "@chakra-ui/core";
+import banner from "../assets/banner.jpg";
+
+const SignUpStyle = styled.div`
```

Comment:
Not sure, but maybe we should split styles and components into different files. Something to ask the TL.

Reply:
Yes, let's talk about it at End of Day wrapup